.github/workflows/main.yml

```diff
@@ -43,14 +43,16 @@ jobs:
     strategy:
       matrix:
         python-version: ${{ fromJSON(needs.Envvars.outputs.version_matrix) }}
-        task: [Typecheck, Lint, Yapf, Test]
+        task: [Typecheck, Lint, Ruff, Yapf, Test]
         include:
           - task: Typecheck
             cmd: pytype -j auto .
           - task: Lint
-            cmd: pylint --rcfile .pylintrc --recursive yes .
+            cmd: pylint --rcfile pyproject.toml --recursive yes .
+          - task: Ruff
+            cmd: ruff check
           - task: Yapf
-            cmd: yapf . -drp
+            cmd: yapf --diff --recursive --parallel .
```

Comment:
This doesn't seem to be relevant to the stated change?

Reply:
This is documentation for readers like me who do not use yapf regularly.  If I had to look up these options then it is likely that other contributors will as well.  Leave the campsite cleaner than you found it.